smoketest/harness/twilioassertionsms.go

```diff
@@ -0,0 +1,21 @@
+package harness
+
+import (
+	"github.com/target/goalert/devtools/mocktwilio"
+)
+
+type twilioAssertionSMS struct {
+	*twilioAssertionDevice
+	*mocktwilio.SMS
+}
+
+var _ ExpectedSMS = &twilioAssertionSMS{}
+
+func (sms *twilioAssertionSMS) ThenReply(body string) SMSReply {
+	// TODO: error here with T context
```

Comment:
out of scope or outstanding?

Reply:
It is using `sms.t` to log the error now -- just needed to clean up the TODO comment.